dapr/clients/grpc/_response.py

```diff
@@ -744,3 +745,102 @@ def __init__(
     def status(self) -> TopicEventResponseStatus:
         """Gets the status."""
         return self._status
+
+
+class UnlockResponseStatus(Enum):
+    success = api_v1.UnlockResponse.Status.SUCCESS
+    '''The Unlock operation for the referred lock was successful.'''
+
+    lock_does_not_exist = api_v1.UnlockResponse.Status.LOCK_UNEXIST
+    ''''The unlock operation failed: the referred lock does not exist.'''
+
+    lock_belong_to_others = api_v1.UnlockResponse.Status.LOCK_BELONG_TO_OTHERS
```

Comment:
We do. They are defined in https://github.com/dapr/dapr/blob/1fdc9a151956f061677258479965af61cb34ddf3/dapr/proto/runtime/v1/dapr.proto#L589-L595.  As this API is still rated as Alpha it might be worth changing this _now_. It would required updates to JS and DotNet SDKs but nothing major,

From my side, I fixed the grammar on the python enum ;)